apps/web/package.json

```diff
@@ -77,6 +77,7 @@
     "@sentry/nextjs": "^7.52.1",
     "@snapshot-labs/snapshot.js": "^0.4.40",
     "@tanstack/react-query": "^5.28.4",
+    "@usdv/usdv-widget": "^0.9.8",
```

Comment:
this package is not open-sourced

Reply:
sry what do you mean? You mean the code is minified?

Reply:
Disclaimer added